packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Activate and enable
+---
+
+The Registration Inline Hook must be set up and activated within your Okta Developer Console and enabled in the self-registration page.
+
+To set up and activate the Registration Inline Hook:
+
+1. Navigate to the **Workflow** > **Inline Hooks** page.
+2. Click **Add Inline Hook** and select **Registration** from the drop-down menu.
+3. Add a name for the hook (in this example, "Guide Registration Hook Code").
+4. Add your external service URL, including the endpoint. For example, use your Glitch project name with the endpoint:  `https://your-glitch-projectname.glitch.me/registrationHook`.
+5. Include authentication field and secret. In this example:
+
+    - **Authentication Field** = `authorization`
+    - **Authorization Secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+6. Click **Save**.
+
+The Registration Inline Hook is now set up with a status of active.
+
+> **Note:** You can also setup an inline hook using an API. See [Inline Hook Setup](/docs/reference/api/inline-hooks/) for further information.
```

Comment:
Instead of "Inline Hook Setup", can we call this "Inline Hooks Management API" as the link text?